packages/bot-web-ui/src/components/dashboard/tutorial-tab/faq-content.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Text, Accordion } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
```

Comment:
can we move the default imports over named imports @farabi-deriv

Reply:
sure. done.